cognite/extractorutils/__init__.py

```diff
@@ -16,5 +16,5 @@
 Cognite extractor utils is a Python package that simplifies the development of new extractors.
 """
 
-__version__ = "5.0.1"
+__version__ = "5.0.2"
```

Comment:
As this is a non-breaking feature update it should be a semimajor version, so we should be on `5.1.0`